source/ed/connectors/gtfs_parser.cpp

```diff
@@ -388,85 +388,113 @@ bool StopsGtfsHandler::parse_common_data(const csv_row& row, T* stop) {
     return true;
 }
 
-StopsGtfsHandler::stop_point_and_area StopsGtfsHandler::handle_line(Data& data, const csv_row& row, bool) {
+nm::StopArea* StopsGtfsHandler::build_stop_area(Data& data, const csv_row& row) {
```

Comment:
Returning a `unique_ptr<nm::StopArea>` here would be much better ! That would prevent you from leaking like an old  person....  
Same applies for `StopsGtfsHandler::build_stop_point()`

Reply:
returning a `unique_ptr` require adding 3 or 4 `get()` in this method to store the raw pointer where we need it, then a call to `release()` later. If the pointer was to be released we will have invalid pointer stored everywhere.
Using `shared_ptr` in `Data` and `GtfsData` would solve this problem but it's outside the scope of this PR.

Reply:
np